pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -69,9 +69,17 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 			return util.ToAdmissionResponseError(err)
 		}
 
+		// Remove all new lines from both certs for the comparison.
```

Comment:
if we extract the code below to a method named removeAllNewLines or trimAllNewLines (if we go line-by-line) it would be a bit more readable and we won't need this comment ;)

Reply:
ack, extracted to new function.